oracle-plugin/src/e2e-test/resources/errorMessage.properties

```diff
@@ -12,7 +12,7 @@ errorMessageInvalidImportQuery=Import Query select must contain the string '$CON
   \ to 1. Include '$CONDITIONS' in the Import Query
 errorMessageBlankUsername=Username is required when password is given.
 errorMessageInvalidTableName=Error encountered while configuring the stage: 'Error occurred while trying to get schema from database.Error message: 'IO Error: Unknown host specified '. Error code: '17002'. SQLState: '08006''
-errorMessageInvalidSinkDatabase=Error encountered while configuring the stage: 'Error occurred while trying to get schema from database.Error message: 'IO Error: Unknown host specified '. Error code: '17002'. SQLState: '08006''
+errorMessageInvalidSinkDatabase=Error encountered while configuring the stage: 'Error occurred while trying to get schema from database.
```

Comment:
This is not correct, why are we making the error message generic, it should cover the exact error message like before.

Reply:
Changes done. Updated the error messages, not including the dynamic/privacy data if present in error msgs.